### ISSUE TITLE
Keyboard focus indicator is not clearly visible on list options.: A11y_VS Code extension for API Center_APICenter_DefinitionFeature_Keyboard

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/vscode-azureapicenter#109]
### GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #WCAG2.4.7; #GH_VSCodeextensionforAPICenter_Linux_Apr2024; #DesktopApp; #VS Code Extension for API Center; #Keyboard; #Linux; #FTP; 

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Linux- Ubuntu 22.04.4

### Repro Steps:

1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created, and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. List options will appear.
7. Navigate using down/up arrow key and observe that Keyboard focus indicator is not clearly visible on list options.

### Actual Result:
Keyboard focus indicator is not clearly visible on list options when we navigate using up/down arrow key.

### Expected Result:
Keyboard focus indicator should be clearly visible on list options when we navigate using up/down arrow key.

### User Impact:
People with low vision often have difficulty perceiving tab focus that have insufficient contrast if Luminosity contrast ratio of visual focus indicator fails to meet the required ratio of 3:1. This can be exacerbated if the person has a color vision deficiency that lowers the contrast even further. Providing a lightness difference of 3:1 or greater can make these items more distinguishable when the person does not see a full range of colors.

### Attachment:

**Comment by formulahendry** [comment]
External issue from Tree View UI of VS Code: https://github.com/microsoft/vscode

**Comment by Yash14j** [comment]
@formulahendry As per the C+AI accessibility testing external bug process primary team need to file the bug to VSCode team & share us the external bug ID.

**Comment by formulahendry** [comment]
Opened bug in vs code repo: https://github.com/microsoft/vscode/issues/212318

**Comment by AnKushSingh05** [comment]
#FirstParty:Bug212318; #External: #External:VS Code;
